web/Services/AdminService.cs

```diff
@@ -98,22 +97,33 @@ private async Task<List<UserDbModel>> GetPendingAdmins()
         return pendingAdmins;
     }
 
-    public async Task<List<LandlordDbModel>> GetLandlordDisplayList(string approvalStatus)
+    public async Task<List<LandlordDbModel>> GetLandlordList(string[] filterArray)
     {
-        return approvalStatus switch
+        var landlordQuery = from landlords in _dbContext.Landlords select landlords;
+        for (var currentFilterNo = 0; currentFilterNo < filterArray.Length; currentFilterNo++)

```

Comment:
I know this is similar to below, but I think it would be simpler to have a view model object with bool? properties for approval status and assigned, rather than having an array where you just have to know what filter is what

Also I _think_ `from landlords in _dbContext.Landlords select landlords` is the same as just `_dbContext.Landlords`

Reply:
Given that we're unlikely to be filtering the landlords by many (if any) other properties, that probably is better - it will be easier to keep track of, and won't cause a load of tedious repetition.